### ISSUE TITLE
Update AddinDisco to update module versions

### ISSUE THREAD

**Issue by augustoproiete** [cake-contrib/Cake.AddinDiscoverer#181]
> _Very low priority given that Cake.DotNetTool.Module has been integrated into the Cake repo_

AddinDisco submitted https://github.com/cake-contrib/Cake.Recipe/pull/799 updating versions of addins that Cake.Recipe relies on, and didn't update the version Cake.DotNetTool.Module [because it's not an addin](https://github.com/cake-contrib/Cake.Recipe/pull/799#issuecomment-787473908).

https://github.com/cake-contrib/Cake.Recipe/blob/f7b0621645e8c8e9e709ea6e12d7957e781675c3/Cake.Recipe/Content/modules.cake

**Comment by Jericho** [comment]
This is blocked by #172
